bitnami/redis-cluster/7.2/debian-11/prebuildfs/opt/bitnami/scripts/libbitnami.sh

```diff
@@ -45,9 +45,9 @@ print_image_welcome_page() {
     local github_url="https://github.com/bitnami/containers"
 
     log ""
```

Comment:
Are there any cons to doing this?
```suggestion
    info ""
```

Reply:
Than its not an empty line. Its a line with the log severity. That was the reason I didn't change it. 
But is doesn't metter. So I can do that